.github/workflows/lint.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       - name: Check immutable yarn.lock
         run: git diff --exit-code
       - name: Lint
-        run: yarn lint:ci
+        run: |
+          echo "::add-matcher::.github/workflows/cspell-problem-matcher.json"
```

Comment:
As an aside, we haven't seen eslint error matchers for a long time. Maybe that's a problem with our setup-node config?